packages/backend/src/invoice/service.ts

```diff
@@ -50,13 +51,16 @@ async function getInvoice(
 
 async function createInvoice(
   deps: ServiceDependencies,
-  userId: string
+  accountId: string,
+  description: string,
+  expiresAt?: Date
 ): Promise<Invoice> {
-  const user = await deps.userService.get(userId)
-  const subAccount = await deps.accountService.createSubAccount(user.accountId)
+  const subAccount = await deps.accountService.createSubAccount(accountId)
   return Invoice.query(deps.knex).insertAndFetch({
-    userId: userId,
-    accountId: subAccount.id,
+    accountId: subAccount.superAccountId,
```

Comment:
```suggestion
    accountId,
```
I'm guessing it would be a problem if `subAccount.superAccountId` isn't the `accountId` passed to `createInvoice`, so maybe also plan to add a check in `createSubAccount` when it's actually calling the accounts service.

Reply:
I've added an assert now to check that the are equal in `createSubAccount`